CONTRIBUTING.md

```diff
@@ -1,8 +1,23 @@
+#
+# Copyright (c) 2021, 2022 by Delphix. All rights reserved.
+#
+
 ## Contributing
  1. Fork the project.
- 1. Make your bug fix or new feature.
- 1. Add tests for your code.
- 1. Send a pull request.
+ 2. Create a virtual environment with `Python 3.8`
+    - Pre Requisite is that you have `Python 3.8` already installed 
+    - You can the run the below command to check that
+ 
```

Comment:
Also for Python 3.X we need not go back to Python 3.5 + since the support for these are being removed from most of the packages
hence we will mentioned 3.7+ to be safe

Reply:
Added a link to README to avoid duplication of content